File: ts/wam/3486.ts
export const WamKeepInChatNux = 3486 as const

export interface KeepInChatNuxEvent {
  /** field 1, wire `chat_ephemerality_duration` */
  chatEphemeralityDuration?: number
  /** field 2, wire `kic_nux_action_name` */
  kicNuxActionName?: KICNUXACTIONNAMETYPE
  /** field 3, wire `thread_id` */
  threadId?: string
  /** field 4, wire `trigger` */
  trigger?: TRIGGERTYPE
}

File: go/wam/3486.go
const WamKeepInChatNux = 3486 // channel: regular

type KeepInChatNuxEvent struct {
	ChatEphemeralityDuration int64 `wam:"chat_ephemerality_duration"` // field 1
	KicNuxActionName KICNUXACTIONNAMETYPE `wam:"kic_nux_action_name"` // field 2
	ThreadId string `wam:"thread_id"` // field 3
	Trigger TRIGGERTYPE `wam:"trigger"` // field 4
}